src/app/legacy/containers/StoryPromo/useCombinedClickTrackerHandler.js

```diff
@@ -6,13 +6,13 @@ const useCombinedClickTrackerHandler = eventTrackingData => {
   const blockData = path(['block'], eventTrackingData);
   const linkData = path(['link'], eventTrackingData);
   const optimizely = path(['block', 'optimizely'], eventTrackingData);
-  const handleBlockLevelClick = useClickTrackerHandler({
+  const { onClick: handleBlockLevelClick } = useClickTrackerHandler({
```

Comment:
Not sure what this means - will click tracking be enabled on lite for StoryPromos?